brain-marks/Settings/Views/AppIconListView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  AppIconListView.swift
+//  brain-marks
+//
+//  Created by Koty Stannard on 10/6/22.
+//
+
+import SwiftUI
+
+struct AppIconListView: View {
+    
+    @EnvironmentObject var appIconSettings: AppIconSettings
+    
+    var body: some View {
+        Form {
+            Picker("App Icons", selection: $appIconSettings.iconIndex) {
+                ForEach(appIconSettings.icons.indices, id: \.self) { index in
+                    IconRow(icon: appIconSettings.icons[index])
+                        .tag(index)
+                }
+            }
+            .pickerStyle(.inline)
+            .onChange(of: appIconSettings.iconIndex) { newIndex in
+                guard UIApplication.shared.supportsAlternateIcons else {
+                    print("Application does not support alternate icons.")
+                    return
+                }
+                
+                let currentIndex = appIconSettings.icons.firstIndex(where: { icon in
+                    return icon.iconName == appIconSettings.currentIconName
+                }) ?? 0
+                
+                guard newIndex != currentIndex else { return }
+                
+                let selectedIcon = appIconSettings.icons[newIndex].iconName
+                UIApplication.shared.setAlternateIconName(selectedIcon) { error in
+                    if let error = error {
+                        print(error.localizedDescription)
```

Comment:
Can you use logger instead of print?
Create a new static property in `Logger+Ext.swift`:
`static let appIcon = Logger(subsystem: subsystem, category: "appIcon")`

You'll have to import os.log and use `Logger.appIcon.error("message here")`

Reply:
@mikaelacaron I have a merge conflict with the dev branch and I need to pull changes into my feature branch to get the Logger+Ext file, but I am still very new to version control stuff with open source projects so I'm not quite sure what to do here. I tried doing a regular pull request but the conflict broke the project & I had to undo my commit.

Reply:
* First update your fork, there's a button that says "sync with origin"
   * This will sync your `dev` branch in GitHub with the `dev` branch on my GitHub
* Make sure you are on `dev` on your local repo (the code on your computer)
* Pull these changes into the `dev` branch
* Checkout your branch `130-alt-icons`
* Merge `dev` into `130-alt-icons`
   * This will create a merge conflict that you will have to fix


If you have trouble doing those steps, let me know!